example/vectorAdd/src/main.cpp

```diff
@@ -73,21 +73,41 @@ class VectorAddKernel
     }
 };
 
+template<class A, class B>
+#if defined(ALPAKA_ACC_CPU_BT_OMP4_ENABLED)
+using DefaulAcc = alpaka::acc::AccCpuOmp4<A,B>;
```

Comment:
I guess there is a typo and `DefaulAcc` should be `DefaultAcc` everywhere?

Reply:
Yes. Fixed.